src/js/components/Form/stories/Select.js

```diff
@@ -1,24 +1,32 @@
-import React, { useState } from 'react';
+import React, { useCallback, useState } from 'react';
 import { storiesOf } from '@storybook/react';
 
-import { grommet, Box, FormField, Select, Grommet } from 'grommet';
-import { allOptions } from './data';
+import { grommet, Box, Form, FormField, Select, Grommet } from 'grommet';
 
-const FormFieldSelect = props => {
-  const [value, setValue] = useState('');
+const options = [
+  { label: 'option 1', value: 1 },
+  { label: 'option 2', value: 2 },
+  { label: 'option 3', value: 3 },
+];
+
+const FormFieldSelect = () => {
+  const [value, setValue] = useState({});
+  const onChange = useCallback(nextValue => setValue(nextValue), []);
 
   return (
     <Grommet theme={grommet}>
       <Box align="center" pad="large">
-        <FormField label="Label" htmlFor="select" {...props}>
-          <Select
-            id="select"
-            placeholder="placeholder"
-            options={allOptions}
-            value={value}
-            onChange={({ option }) => setValue(option)}
-          />
-        </FormField>
+        <Form value={value} onChange={onChange}>
```

Comment:
Let's support the old version of this story (just FormField and no Form) on a test for backward compatibility.

Reply:
Since the prior story didn't even have `<Form/>` in it, I put the old story content in `FormField/stories/Select.js` and kept this one as the `Form` version.